libr/anal/fcn.c

```diff
@@ -1337,7 +1337,8 @@ R_API int r_anal_fcn(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 len, int r
 	}
 	fcn->maxstack = 0;
 	if (fcn->cc && !strcmp (fcn->cc, "ms")) {
-		fcn->stack = fcn->maxstack = 0x28; // Shadow store for the first 4 args + Return addr
+		// Probably should put this on the cc sdb 
+		fcn->stack = fcn->maxstack = fcn->reg_save_area = 0x28; // Shadow store for the first 4 args + Return addr
```

Comment:
shorten this comment by using const `const int shadow_store = 0x28; // first 4 args+retaddr`